Formula/apache-archiva.rb

```diff
@@ -22,6 +22,8 @@ def post_install
     cp_r libexec/"conf", var/"archiva"
   end
 
+  plist_options :manual => "ARCHIVA_BASE=#{HOMEBREW_PREFIX}/var/archiva #{HOMEBREW_PREFIX}/opt/apache-archiva/bin/archiva console"
```

Comment:
Running the `archiva` without prefixing the path raises an error. 

```
› ARCHIVA_BASE=/usr/local/var/archiva archiva console
Running Apache Archiva...
FATAL  | wrapper  | Unable to resolve the full path of the configuration file, /usr/local/Cellar/apache-archiva/2.2.0/conf/wrapper.conf: No such file or directory
```

wrapper.conf is searched in another location due to the way `archiva`'s wrapper is defined

```sh
...
# discover BASEDIR
BASEDIR=`dirname "$0"`/..
BASEDIR=`(cd "$BASEDIR"; pwd)`
ls -l "$0" | grep -e '->' > /dev/null 2>&1
if [ $? = 0 ]; then
  #this is softlink
  _PWD=`pwd`
  _EXEDIR=`dirname "$0"`
  cd "$_EXEDIR"
  _BASENAME=`basename "$0"`
  _REALFILE=`ls -l "$_BASENAME" | sed 's/.*->\ //g'`
   BASEDIR=`dirname "$_REALFILE"`/..
   BASEDIR=`(cd "$BASEDIR"; pwd)`
   cd "$_PWD"
fi



# Wrapper
WRAPPER_CMD="./wrapper"
WRAPPER_CONF="$BASEDIR/conf/wrapper.conf"
...
```